draft-bradleylundberg-cfrg-arkg.md

```diff
@@ -540,10 +537,6 @@ BL-Blind-Secret-Key(sk, tau) -> sk_tau
     sk_tau = sk_tau_tmp
```

Comment:
Same here, generation of sk_tau is not always like this. Refer to invoking the suite specific key generation routine.